packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -88,14 +88,20 @@ export default /* @ngInject */ ($stateProvider) => {
         VpsService.getTaskInError(serviceName).then((tasks) =>
           head(filter(tasks, { type: 'migrate' })),
         ),
+      service: /* @ngInject */ ($http, serviceInfo) =>
+        $http
+          .get(`/services/${serviceInfo.serviceId}`)
+          .then(({ data }) => data),
 
       configurationTile: /* @ngInject */ (
         availableUpgrades,
         catalog,
+        service,
         stateVps, // from apiv6
         vps, // from 2api
       ) => ({
         upgrades: VpsConfigurationTile.setVps(vps, stateVps.model)
+          .setCurrentService(service)
```

Comment:
it's a better practice to have immutable service but since it's a fix and VpsConfigurationTile already has the mutable setVps method i guess this should be addressed in a refactoring PR

Reply:
@frenautvh I agree this is not a good practice, it was done that way because it was already the case. As the impact is quite huge, more time would be required for a refactor